docs/visual-basic/language-reference/operators/nameof.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "NameOf operator - Visual Basic"
```

Comment:
Oops this wasn't added to the TOC - I think all new topics need to go into a TOC.